metricbeat/module/vsphere/datastore/_meta/fields.yml

```
@@ -4,32 +4,66 @@
     datastore
   release: ga
   fields:
-    - name: name
-      type: keyword
-      description: >
-        Datastore name
-    - name: fstype
-      type: keyword
-      description: >
-        Filesystem type
-    - name: capacity.total.bytes
+    - name: capacity.free.bytes
       type: long
       description: >
-        Total bytes of the datastore
+        Free bytes of the datastore.
       format: bytes
-    - name: capacity.free.bytes
+    - name: capacity.total.bytes
       type: long
       description: >
-        Free bytes of the datastore
+        Total bytes of the datastore.
       format: bytes
     - name: capacity.used.bytes
       type: long
       description: >
-        Used bytes of the datastore
+        Used bytes of the datastore.
       format: bytes
     - name: capacity.used.pct
       type: scaled_float
       description: >
-        Used percent of the datastore
+        Used percent of the datastore.
       format: percent
+    - name: fstype
+      type: keyword
+      description: >
+        Filesystem type
+    - name: host.count
+      type: long
+      description: >
+        Number of hosts.
+    - name: iops
+      type: long
+      description: >
+        Storage I/O Control aggregated IOPS.
+    - name: name
+      type: keyword
+      description: >
+        Datastore name
+    - name: read.bytes
+      type: long
+      description: >
+        Rate of reading data from the datastore.
+      format: bytes
+    - name: read.latency.total.ms
+      type: double
+      description: >
+        Average amount of time for a read operation from the datastore.
+    - name: status
+      type: keyword
+      description: >
+        Status of the datastore.
+    - name: vm.count
+      type: long
+      description: >
+        Number of VMs.
+    - name: write.bytes
+      type: long
+      description: >
+        Rate of writing data to the datastore.
+      format: bytes
+    - name: write.latency.total.ms
+      type: double
```

I think this should be `long`.
```suggestion
      type: long
```